walletlibrary/src/test/java/com/microsoft/walletlibrary/did/sdk/identifier/resolvers/MockRootOfTrustResolverFailure.kt

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved
+
+package com.microsoft.walletlibrary.did.sdk.identifier.resolvers
+
+import com.microsoft.walletlibrary.did.sdk.identifier.models.identifierdocument.DidMetadata
```

Comment:
why create three different objects for this? Can you pass the expected result in the constructor?